CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+## [3.1.0](https://github.com/commercetools/merchant-center-application-kit/compare/v3.0.1...v3.1.0) (2018-11-28)
+
+#### ✍️ Type: Documentation
+
+- [#145](https://github.com/commercetools/merchant-center-application-kit/pull/145) chore: update docs to include build step ([@montezume](https://github.com/montezume))
+
+#### 💅 Type: Enhancement
+
+- `application-shell`
+  - [#143](https://github.com/commercetools/merchant-center-application-kit/pull/143) feat(test-utils): run tests agains actual ApolloProvider ([@pa3](https://github.com/pa3))
+  - [#144](https://github.com/commercetools/merchant-center-application-kit/pull/144) Fix double dependency bundling ([@tdeekens](https://github.com/tdeekens))
+
+#### 🚀 Type: New Feature
+
+- `application-shell`, `i18n`
+  - [#142](https://github.com/commercetools/merchant-center-application-kit/pull/142) feat(app-shell/navbar): add menu item for custom apps ([@emmenko](https://github.com/emmenko))
+
+#### Committers: 4
+
+- Fedor Kulikov ([@pa3](https://github.com/pa3))
+- Malcolm Laing ([@montezume](https://github.com/montezume))
```

Comment:
have we decided if we are adding this section or not? Because it seems like half the time we do, half we don't.

Reply:
Please remove it, it’s not necessary

Reply:
Sure. Will not add it from now on.